### ISSUE TITLE
EmptyClassBlock should consider comment as "body" (via option?)

### ISSUE THREAD

**Issue by TWiStErRob** [arturbosch/detekt#1756]
## Expected Behavior of the rule
```kotlin
class Foo {
    // this is a comment about why Foo doesn't have a body (implementation detail)
}
```
the above shouldn't report `EmptyClassBlock`, IMO

## Context
Documentation like this belongs to the class, but it's not KDoc as it's not the description of a public API.
Without the class body braces the comment would be just dangling in space, and for example Move (class) refactor wouldn't consider it to be part either and would leave it where it is.

```kotlin
class Foo
// just a random comment unrelated to Foo
```

**Comment by cortinico** [comment]
Agree, here the fix: #1758
